seed/models/tax_lot_properties.py

```diff
@@ -117,6 +117,8 @@ def get_related(cls, object_list, columns):
                 # Do not make these timestamps naive. They persist correctly.
                 related_dict['updated'] = related_view.property.updated
                 related_dict['created'] = related_view.property.created
+                # Replace the enumerations
+                related_dict['analysis_state'] = related_view.state.get_analysis_state_display()
```

Comment:
So was `analysis_state` already a key here somewhere and you are now overriding it with this new `_display` method?

Reply:
it exists when calling model_to_dict. I originally tried hooking in the PropertyStateSerializer but it would require a bit of customization to work right. Ideally that is what we will need to do sooner rather than later.

Reply:
I figured there was some serializer action automatically transforming the members; makes sense.